packages/web/hooks/ui-config/use-asset-info-config.ts

```diff
@@ -281,19 +289,11 @@ export class ObservableAssetInfoConfig {
     return this._historicalRange;
   }
 
-  constructor() {
+  constructor(denom: string, coinMinimalDenom?: string) {
```

Comment:
BTW it's possible to remove a few LOC elsewhere in the class if you add `readonly` in front of each param, then they can be accessed directly via `this.denom` (without having to set `this.denom = denom`)